airbyte-webapp/src/components/ui/CheckBox/CheckBox.tsx

```diff
@@ -10,9 +10,10 @@ type CheckBoxSize = "lg" | "sm";
 export interface CheckBoxProps extends InputHTMLAttributes<HTMLInputElement> {
   indeterminate?: boolean;
   checkboxSize?: CheckBoxSize;
+  tabIndex?: number;
 }
 
-export const CheckBox: React.FC<CheckBoxProps> = ({ indeterminate, checkboxSize = "lg", ...inputProps }) => {
+export const CheckBox: React.FC<CheckBoxProps> = ({ indeterminate, checkboxSize = "lg", tabIndex, ...inputProps }) => {
```

Comment:
This is specifically to allow us to remove the checkbox from the regular tabbing flow (ie: when it's inside of a dropdown :) )